snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -256,16 +258,7 @@ func (b *bootstrapper) Connected(
 		return err
 	}
 
-	if err := b.StartupTracker.Connected(ctx, nodeID, nodeVersion); err != nil {
-		return err
-	}
-
-	if b.started || !b.StartupTracker.ShouldStart() {
-		return nil
-	}
-
-	b.started = true
-	return b.Startup(ctx)
+	return b.StartupTracker.Connected(ctx, nodeID, nodeVersion)
```

Comment:
q: we skip checks on connected stake that are carried out in common bootstrapper Startup. Why is this? My guess is that we don't need any voting round to now what the stop vertex is: we either use the hard-coded one or we pick a stop vertex already available locally. So no need to wait for a validator majority to connect. We can download vertexes from any peer (even non validator). Is this correct?
Ps obviously we still want to track connected peers since startupTracker is used later on by the linearized bootstrapper

Reply:
that's correct